dashboard/client/src/pages/FunctionLogPage.jsx

```diff
@@ -25,18 +25,23 @@ export class FunctionLogPage extends Component {
       log: '',
       functionName,
       user,
-      fetchError: false
+      fetchError: false,
+      numLines: 0,
+      since: 1
```

Comment:
since default might want to be 0, so it fetches the numLines by default

Reply:
Sure, will make that change